### DISCUSSION TITLE
Ender S1 display on S1 Pro possible?

### DISCUSSION THREAD

**Discussion by neumo** [mriscoc/Ender3V2S1#161] [Q&A] [answered]
Hi, is it possible to get the S1 display working on the S1 Pro printer?
This as basis to get the professional firmware usable for the S1 Pro too.
Any idea appreciated!

**Comment by mriscoc** [comment]
Yes, it is possible, see the picture from Nathan Brooks:
![image](https://user-images.githubusercontent.com/2745567/169552546-4f614daa-0d01-410a-9dc8-a39c679a73c7.png)

**Comment by Nbrooks639** [comment, marked as answer]
yep im here bud, it does work and works great, get rid of that touch screen. ok first you need to obviously 
find a screen, also you will need the ribbon cable for the other screen because the touch screen cable is different.

**Reply by neumo** [reply]
Thanks. Not cheap but I saw the Display at AliExpress.
After updating the Firmware and the Display ist should Work at the Pro. I guess it is the normal way like for the S1.  Right?
The higher temperatures of the hotend and heating bed can be configured somewhere?

**Reply by Nbrooks639** [reply]
you can get the display from amazon. its just a regular ender 3 v2/ender 3 s1 display, the ribbon cable comes with it.  and yes update just like regular s-1, temp can be configured.

**Reply by neumo** [reply]
Thanks for all the infos/reply. I got the display today, made the update. 
My S1 Pro has the STM32F4 board and after moving the bin file in a subfolder STM32F4_UPDATE the update was done.
Did most of the calibrations and now the first print is running.
Very good work - very impressing!

**Reply by hayden-t** [reply]
just ordered a screen of aliexpress, $25 usd ! octoprint works with runout well on stock, but would like host actions and other new stuff so will try this

**Reply by mm108** [reply]
> Thanks for all the infos/reply. I got the display today, made the update. My S1 Pro has the STM32F4 board and after moving the bin file in a subfolder STM32F4_UPDATE the update was done. Did most of the calibrations and now the first print is running. Very good work - very impressing!

hi @neumo, just checking if everything's working smooth after your update as I just got a S1 Pro and thinking of using this firmware. Any tips that you could share - or any problems that you encountered so far? Thank you so much. Appreciate any tips/input. Cheers.

**Reply by hayden-t** [reply]
screen works for me, no problems ! firmware is great

**Reply by mm108** [reply]
> screen works for me, no problems ! firmware is great

Cool, thanks @hayden-t

**Reply by neumo** [reply]
Still everything is fine. Take care to get a display with suitable cable.

**Reply by mm108** [reply]
Thanks for the affirmation. I ordered a S1 display (non touch / with knob ). Hope everything works as intended. 

The S1 pro machine is great but firmware sucks!

**Reply by mattkerrison** [reply]
@Nbrooks639 I'm looking at picking up an S1 Pro, I already own a Ender 3V2, am I correct in thinking I can just use the ribbon cable and display from my v2 on the S1 Pro? I don't use my Ender 3 V2 display anymore as using octoprint and octodash (likely will with S1 Pro too, but want to be able to set machine settings initially)

**Reply by hayden-t** [reply]
correct

**Reply by Nbrooks639** [reply]
> @Nbrooks639 I'm looking at picking up an S1 Pro, I already own a Ender 3V2, am I correct in thinking I can just use the ribbon cable and display from my v2 on the S1 Pro? I don't use my Ender 3 V2 display anymore as using octoprint and octodash (likely will with S1 Pro too, but want to be able to set machine settings initially)

yes you sure can. just need to unplug the touch screen cable and plug the rotary knob screen cable into the port next to it. very simple and pro firm is a go.
